test-case-reporting/test/test_result_record.test.ts

```diff
@@ -390,5 +390,20 @@ describe('TestResultRecord', () => {
         'gets the recent builds in reverse chronological order(most recent first)'
       );
     });
+
+    describe('getTestCasesSortedByStat', () => {
+      // TODO(#975): Add unit tests for getTestCasesSortedByStat
+      it.todo('gets the test cases, sorted by skipped percentage');
+
+      it("doesn't allow for bad columns", async () => {
```

Comment:
"rejects attempts to sort by illegal columns"

Reply:
Done.